react-effects-and-fetch_pokemon-api-with-paging/src/components/PokemonList.jsx

```diff
@@ -1,15 +1,15 @@
 import { useEffect, useState } from "react";
 import "./PokemonList.css";
 
-
 export default function PokemonList() {
   const [pokemon, setPokemon] = useState([]);
+  const [page, setPage] = useState(1);
```

Comment:
it's offset not page, so should be adding 20 every time - you are only skipping one pokemon at a time here